src/org/labkey/test/tests/CrossSiteScriptingForDeleteTest.java

```diff
@@ -0,0 +1,104 @@
+package org.labkey.test.tests;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.DailyB;
+import org.labkey.test.components.ext4.Window;
+import org.labkey.test.components.html.BootstrapMenu;
+import org.labkey.test.pages.reports.ManageViewsPage;
+
+import java.util.Collections;
+import java.util.List;
+
+@Category({DailyB.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 3)
+public class CrossSiteScriptingForDeleteTest extends BaseWebDriverTest
+{
+    protected static final String PROJECT_NAME = "CrossSiteScriptingForDeleteTest";
+    protected static final String REPORT_NAME = "<span onclick='alert(\"fooled you!\")'>mouseover me, it's totally safe</span>";
```

Comment:
Use an injection string that doesn't require direct interaction. It gives us a chance to catch something we aren't specifically checking for.
```suggestion
    protected static final String REPORT_NAME = BaseWebDriverTest.INJECT_CHARS_1;
```

Reply:
Done.